class.jetpack-jitm.php

```diff
@@ -321,7 +321,7 @@ function backups_updates_msg() {
 			<a href="#" data-module="vaultpress" class="dismiss"><span class="genericon genericon-close"></span></a>
 
 			<div class="jp-emblem">
-				<?php echo self::get_jp_emblem(); ?>
+				<?php echo Jetpack::get_jp_emblem(); ?>
```

Comment:
I think the changes that I proposed would make any future rewrite much easier and shouldn't be that hard to implement.